src/components/Input/types.ts

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { InternalInputColors, InternalInputTypes, InternalInputVariants } from "../internal/InternalInput";
+
+export interface DeprecatedInputProps {
+  name: string,
+  label: string;
+  id?: string,
+  type?: InternalInputTypes,
+  placeholder?: string,
+  variant?: InternalInputVariants,
+  prefix?: string,
+  suffix?: string,
+  maxLength?: string | number,
+  autoComplete?: string,
+  optional?: boolean,
+  color?: InternalInputColors,
+  helpText?: React.ReactNode,
+  disabled?: boolean,
+  pasteAllowed?: boolean,
+  isValid?: boolean,
+  testId?: string;
+  describedBy?: string,
+  value?: string,
+  onChange?: React.ChangeEventHandler<HTMLInputElement>,
+  errors?: React.ReactNode[],
+  error?: string | string[],
```

Comment:
Do we need `errors` and `error`? The deprecated `Input` doesn't have these props.

Reply:
We don't need `errors` I will remove it